### DISCUSSION TITLE
Energy of carbon ions at different water depths

### DISCUSSION THREAD

**Discussion by Yuri-HubL** [e0404/matRad#664] [Q&A]
Dear Developer

I was wondering if you could provide information on the energy of carbon ions at different water depths.Or did you use some kind of analytical formula to calculate the energy of the carbon ions at different depths?

To give you a better idea of what I mean, let me explain in detail with the image below. First we can see the first energy (115.23MeM/u) with its depth points recorded in the column "Depth". The data I want to get is the energy at these different sites.

I am looking forward to your reply, which will help me a lot.
your sincerely,
Yuri

**Comment by Yuri-HubL** [comment]
![carbon energy](https://github.com/e0404/matRad/assets/72288469/18f143dd-a8d4-47b7-a654-745016b27ef6)

**Reply by wahln** [reply]
Please note that this base data is for using pencil-beam algorithms. Thus this database contains _dose kernels_ in depth, and you don't have access to energy of the primary particle in depth, not even to the initial energy spectrum (because the algorithm does not need it). You would need to make a Monte Carlo simulation with a carbon ion source parameterization to get such information. See also the answer by @remocristoforetti

**Comment by remocristoforetti** [comment]
Hello Yuri,
I'm not sure if I understand your question correctly.
The `energy` that you see stored in the base data is the initial nominal energy of the carbon beam. What is stored in `depths`, are just the water equivalent depths at which all the other quantities (`Z`, `sigma`, etc) are stored.

If by _energy at these different sites_ you mean the energy spectrum of the beam sampled at different depths, that information is not stored in the base data and cannot be obtained with `matRad` at the moment.

I remain at your disposal,
Remo

**Reply by Yuri-HubL** [reply]
Hi Remo,

Thank you very much for your reply. Your understanding is correct. What I'm asking for here is the spectrum at different depths.

Although matRad cannot provide this data, do you have any suggestions or other ways for me to obtain the energy spectrum at different depths?

I am looking forward to your reply.

best,
Yuri

**Reply by Yuri-HubL** [reply]
Hi Remo，

Writing this letter, I would like to ask further questions. We know that the LET of carbon ions at different equivalent water depths is different, but few people discuss the MeV/u value at different equivalent water depths. I wonder why there is less discussion about this?

As shown in the figure below, how should I determine the MeV/u values of the three points? Because I'm more interested in MeV over u right now
![image](https://github.com/e0404/matRad/assets/72288469/8a526994-4327-48eb-a3c4-ab71752ef058)

I am looking forward to your reply

your sincerely,
Yuri

**Reply by remocristoforetti** [reply]
Dear Yuri,
the questions you are rising are very interesting and stimulating. However, what you are asking is beyond the scope of `matRad` and the purpose of this forum.
Here we can only answer `matRad` relates issues, and we are happy to help users setup and profiquosly exploit the `matRad` capabilities for their daily reserch.
If you have any further `matRad` related question, feel free to keep asking here.

Thank you for your undersanding.
Best regards,
Remo
